### ISSUE TITLE
Title isn't populated when in a subfolder of a repository

### ISSUE THREAD

**Issue by Eric162** [wisetime-io/vscode-branch-in-window-title#21]
First of all - thank you for this extension - it should be a part of VS code natively!

This line makes an assumption that you're always in the root of a project. In reality for very large monorepos people (like me) probably just open subfolders).

https://github.com/wisetime-io/vscode-branch-in-window-title/blob/14957fe142f1c4ffe7e7b50758a1d18014e79dce/src/extension.ts#L16C69-L16C69

It would be awesome instead of using the workspace root, to instead get the git root using something like:
```ts
function getRepositoryRoot() {
  return resolve(execSync('git rev-parse --show-toplevel').toString().trim());
}
```

I believe this would also solve #16 most likely. I'll take a stab at it in a PR

**Comment by AnrDaemon** [comment]
Should be fixed by #23 for good.

**Comment by vyshane** [comment]
We have released [v0.2.0](https://github.com/wisetime-io/vscode-branch-in-window-title/releases/tag/v0.2.0) which addresses this. Thank you for your contribution @AnrDaemon.
